demos/python_demos/llm_text_generation/compress_model.py

```diff
@@ -31,9 +31,14 @@
 
 parser = argparse.ArgumentParser(description='Script to compress LLM model based on https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot')
 
+supported_languages_list = []
```

Comment:
Is this value needed for anything else other than getting models list below?
If not, maybe we could merge them. If I understand the structure it could look like this:
```
supported_models = [model_name for model in SUPPORTED_LLM_MODELS.values() for model_name in model.keys()]
```

Note that in this case (but also in current version it seems) there is a possibility of duplicates.